package.json

```diff
@@ -41,9 +41,9 @@
 		"webpack": "4 || 5"
 	},
 	"dependencies": {
+		"@cspotcode/source-map-support": "^0.8.1",
```

Comment:
It's not clear to me how this fork is different from the original `source-map-support` so I'm going to revert this.